### PR TITLE
Added chmod endpoint

### PR DESCRIPTION
**Pull request by mrxbox98**

### CODE REVIEW

File: reference/Client-API.yaml [hunk 1]
```diff
@@ -2655,6 +2655,45 @@ paths:
...
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                root:
+                  type: string
+                  description: The root directory. Uses '/' if no directory is provided.
+                files:
```
Comment (TekExplorer) [inline reference/Client-API.yaml:2684]: This array should be in the form of
[Source (top level)](https://github.com/pterodactyl/panel/blob/1.0-develop/app/Http/Controllers/Api/Client/Servers/FileController.php#L235-L243)
[Source (the array)](https://github.com/pterodactyl/panel/blob/1.0-develop/app/Repositories/Wings/DaemonFileRepository.php#L246-L263)
```json
{
  "file": "filename.txt",
  "mode": 777
}
```

File: reference/Client-API.yaml [hunk 2]
```diff
@@ -2655,6 +2655,45 @@ paths:
...
+                  type: array
+                  description: The files to update permission of.
+                  items: {}
+              required:
+                - files
+            examples:
+              Example 1:
```
Comment (TekExplorer) [inline reference/Client-API.yaml:2705]: update the example to line up with the above.

File: reference/Client-API.yaml (hunk 1 above)
Comment (TekExplorer) [reply reference/Client-API.yaml:2684]: Also, all parameters with the exception of `root` are required

**Comment by TekExplorer** [comment]
@mrxbox98 you good?

**Comment by mrxbox98** [comment]
> @mrxbox98 you good?

wut

**Comment by TekExplorer** [comment]
> > @mrxbox98 you good?
> 
> wut

could you resolve these comments so i can merge?

**Comment by mrxbox98** [comment]
> > > @mrxbox98 you good?
> > 
> > 
> > wut
> 
> could you resolve these comments so i can merge?

I cant see any comments

**Comment by TekExplorer** [comment]
thats because i forgot to hit submit
goddammit

**Comment by mrxbox98** [comment]
lol fixed

**Review by TekExplorer — APPROVED** [review]
lgtm

**Pull request merged**